### PR TITLE
Add BHNT 77

### PR DESCRIPTION
**Pull request by ligi**
most info condensed and formatted to our standard format from https://pad.systemli.org/p/bhnt077

cc @gimlet2 @pdp7  @vog @momonala @jaseg @strathausen - thanks for presenting and maybe you can check/add details/links for your projects

we should also think about conserving the rest of the data in the pad - the tags and the flea market stuff is interesting. Not yet sure exactly where to add it - but it should be part of the website IMHO as the pad might/will disappear.
Thanks @Wikinaut & @lucasrangit for the pad - seems like a really good supporting infrastructure for BHNT

### CODE REVIEW

File: _posts/2020-01-28-hacking-2020.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+date: 2020-01-28
+time: '19:15'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/268089955/
+---
+
+* [Vote2p](https://github.com/gimlet2/vote2p) (voting, ipfs, pubsub) by [[Andrey](https://github.com/gimlet2)
+* [FPGA Badge Hack](https://hackaday.io/page/6764-hackaday-supercon-badge-boots-linux-using-sdram-cartridge) - Linux on RISC-V by [Drew Fustini](https://github.com/pdp7)
+* [obfuscated compressor](https://ioccc.org/2019/diels-grabsch1/hint.html) by Volker [Diels-Grabsch](https://njh.eu)
+* [supericon](https://github.com/momonala/supersizeme) by [Mohit Nalavadi](https://github.com/momonala)
```
Comment (momonala) [inline _posts/2020-01-28-hacking-2020.md]:
```suggestion
* [supericon](https://github.com/momonala/supersizeme) by [Mohit Nalavadi](https://github.com/momonala)

* [Super Resolution HTTP API](https://github.com/momonala/supersizeme) by [Mohit Nalavadi](https://github.com/momonala)
```

Comment (momonala) [reply _posts/2020-01-28-hacking-2020.md]: Just a name change. Thanks again for the event! Had a nice time.

Comment (ligi) [reply _posts/2020-01-28-hacking-2020.md]: Applied - thanks - great to hear!

**Comment by pdp7** [comment]
@ligi thanks,

**Comment by Wikinaut** [comment]
@ligi:Idea: best way to archive the pad: export as PDF with a meaningful filename and then checking-in the PDF to github repo.

**Comment by Wikinaut** [comment]
[20200128-bhnt077.pdf](https://github.com/berlin-hack-and-tell/berlinhackandtell.rocks/files/4127559/20200128-bhnt077.pdf)

**Comment by Wikinaut** [comment]
see above

**Comment by vog** [comment]
I like the pad because of the low entry barrier and the real-time collaboration.

However, it is a pity that it doesn't encourage the exact (Markdown) format that we need for the website. Wouldn't it be great if we could just copy&paste the pad into the pull request ... and maybe even generate the pull request automatically, so we just need to review before merge?.

Also, editing a Pad as Markdown instead of WYSIWYG shouldn't add too much of a barrier to the typical BHNT audience, I guess. If any, it might make it even more attractive (with a live preview, of course).

The same goes for the flea market. I'd love to have it in Markdown and just added as a new section below the presentations.

**Comment by ligi** [comment]
True - would be great if the pad could spit out MarkDown so we can include it here the same as with the slides and entries. Would love to avoid the usage of PDF to be honest. MarkDown / plain text is much nicer for a lot of reasons over rendered PDFs that are harder searchable and editable afterwards.

**Comment by Wikinaut** [comment]
@vog @ligi There exists a markdown-export plugin, see https://github.com/ether/ep_markdown. I will contact systemli staff, perhaps they are willing to add this.

**Comment by Wikinaut** [comment]
You can also export that pad as plain text.

**Comment by vog** [comment]
Exporting as Markdown is nice, but often results in messy output as the users have no real control over it.

Is it also possible to *edit* the etherpad as Markdown?

**Comment by Wikinaut** [comment]
Because I think, the pad should remain a low-threshold tool, we should use and leave that as it is. Personally, I hate Markdown - because I prefer MediaWiki-Syntax to it. But this would start an endless discussion about the pros and cons.

**Comment by Wikinaut** [comment]
An alternative to the pad would be to simply open the Wiki function here on Github!

**Comment by ligi** [comment]
with the github wiki we would also have the users to edit markdown - right?
I think the barrier to entry is lower with a pad than the wiki where you need to login with your github account which you might not want e.g. on your phone at BHNT

**Comment by Wikinaut** [comment]
Yes. Pls let's go on with the pad as it is.

**Comment by vog** [comment]
Just to clarify: I'm not arguing in favor of GitHub wiki. It neither works without login, nor does it support realtime preview, nor does it support realtime collaborative editing. It is not suited for the purpose at all.

However, I'm arguing for a pad with Markdown syntax and realtime preview. Just in case someone knows about such a pad, or knows about an extension or similar.

**Comment by vog** [comment]
(The edit window of Stackoverflow comes to mind, but it's not realtime collaborative editing.)

**Pull request merged**